boards/x86/up_squared/doc/up_squared.rst

```diff
@@ -300,6 +299,135 @@ Steps
          WARNING: no console will be available to OS
 
 
+Booting the UP Squared Board over network
+=========================================
+
+Build Zephyr image
+------------------
+
+#. Follow `Build Zephyr application`_ steps to build Zephyr image.
+
+Prepare Linux host
+------------------
+
+#. Follow `Creating a GRUB2 Boot Loader Image from a Linux Host`_ steps
+   to create grub binary.
+
+#. Install DHCP, TFTP servers. For example `dnsmasq`
+
+   .. code-block:: console
+
+      $ sudo apt-get install dnsmasq
+
+#. Configure DHCP server. Configuration for `dnsmasq` is below:
+
+   .. code-block:: console
+
+      # Only listen to this interface
+      interface=eno2
+      dhcp-range=10.1.1.20,10.1.1.30,12h
+
+#. Configure TFTP server.
+
+   .. code-block:: console
+
+      # tftp
```

Comment:
Do you need to be root to do this command?  (You're using a ``#`` prompt).  If so, use ``$ sudo tftp``

Reply:
This comment for the dnsmasq configuration